src/test/wasm_o_compile_known_gcc_test_failures.txt

```diff
@@ -0,0 +1,86 @@
+# Tests which are known to fail from the GCC torture test suite.
```

Comment:
I think this is a good candidate for using the capability introduced in #181. We could run builds with e.g. `assembly` and `object` attributes; most of the fails here would be unconditional (they fail for both), and then the few that are only here would have the `object` attribute on the failure entry.